lib/awful/spawn.lua

```diff
@@ -279,7 +279,7 @@ local function hash_command(cmd, rules)
     cmd = type(cmd) == "string" and cmd or table.concat(cmd, ';')
 
     -- Do its best at adding some entropy
-    local concat_rules = nil
+    local concat_rules
```

Comment:
I would prefer if `nil` remained. We still have some C code, so the convention is usually to initialize a variable when you plan to use it as-is. Also, while there is some of both, initialized nil are use more often.

Reply:
Ok, will take in mind.

Also tell me please how PR fixes should be usually organized. Should I amend commit and do force push or leave chain of commit as is?

Reply:
They'll squash the commit before merge.